content/routing.md

```diff
@@ -32,7 +32,7 @@ meteor add kadira:flow-router
 
 *Snippet: Adding the Flow Router package*
 
-At the time of writing this guide, Flow Router is at version 2.x.
+At the time of writing this guide, [Flow Router](https://github.com/kadirahq/flow-router) is at version 2.x.
```

Comment:
I think you're 100% right on this point - the guide can be the entry point for discovering these patterns, but shouldn't be the location for all of the reference material.

1. Links to original project: definitely should have this
2. Discussions about why we removed auth from the router - I don't think the guide is for historical discussions like this; I'd rather stick to guiding materials
3. Articles from Wikipedia might be relevant when talking about complicated concepts that we can't explain inline. Are there any in the routing guide?